### ISSUE TITLE
redirects?

### ISSUE THREAD

**Issue by nycnoise** [gregsadetsky/nycnoise#375]
is there a way for me to create a redirect to a subpage?

in this specific case, i want to rename the /help page, but i've given that url to enough people that i don't want to break the existing link... any thoughts on how i can keep both /help & /[new thing] functional?

**Comment by nycnoise** [comment]
(**not urgent** cuz i just made a new page, & can add a link at top of old one... but figure this might come up again someday!)

**Comment by gregsadetsky** [comment]
yeah for sure redirects could be a thing, it would involve a new model (old url -> new url) and some view code but yeah!
